app/change_set_persisters/plum_change_set_persister.rb

```diff
@@ -1,6 +1,29 @@
 # frozen_string_literal: true
 class PlumChangeSetPersister
+  class_attribute :registered_handlers
+  self.registered_handlers = {}
+
+  def self.register_handler(action, handler)
+    raise "Invalid action #{action} given. Must be one of #{valid_actions.to_sentence}." unless valid_actions.include?(action)
+    registered_handlers[action] ||= []
+    registered_handlers[action] << handler
+  end
+
+  def self.valid_actions
```

Comment:
Just out of curiosity, why can't these be modified in the same manner as registration handlers?